src/Umbraco.Web.UI.Client/src/packages/documents/documents/collection/action/create-document-collection-action.element.ts

```diff
@@ -56,20 +56,20 @@ export class UmbCreateDocumentCollectionActionElement extends UmbLitElement {
 
 	override async firstUpdated() {
 		if (this._documentTypeUnique) {
```

Comment:
Hi @AndyButland 
I think this needs to check for `this._documentUnique !== undefined`